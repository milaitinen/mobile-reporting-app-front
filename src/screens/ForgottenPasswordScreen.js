import React from 'react';
import { StyleSheet, View, Text, TouchableHighlight } from 'react-native';

export default class ForgottenPasswordScreen extends React.Component {

  handleButtonPress = () => {
      this.props.navigation.goBack();
  };

  render() {
      return (
          <View style={styles.container}>
              <TouchableHighlight style={styles.button} onPress={() => this.handleButtonPress()}>
                  <Text style={styles.text}>Forgotten Password Screen</Text>
              </TouchableHighlight>
          </View>
      );
  }
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: '#fff',
        alignItems: 'center',
        justifyContent: 'center',
    },
    button: {
        backgroundColor: '#4F6D7A',
        width: 250,
        height: 50,
        justifyContent: 'center',
    },
    text: {
        textAlign: 'center',
        color: '#fff',
    }
});