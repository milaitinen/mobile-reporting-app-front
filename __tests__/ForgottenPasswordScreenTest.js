import 'react-native';
import React from 'react';
import ForgottenPasswordScreen from '../src/screens/ForgottenPasswordScreen';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <ForgottenPasswordScreen />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});