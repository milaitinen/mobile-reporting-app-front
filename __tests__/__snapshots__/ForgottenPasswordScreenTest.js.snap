// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        Object {
          "backgroundColor": "#4F6D7A",
          "height": 50,
          "justifyContent": "center",
          "width": 250,
        },
      ]
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#fff",
          "textAlign": "center",
        }
      }
    >
      Forgotten Password Screen
    </Text>
  </View>
</View>
`;
